docs/new_flux_ideas_and_guide.md

```diff
@@ -0,0 +1,985 @@
+
+The combination of a measurement, tags, and a specific field represent an individual time series, which are an
+ordered list of time, value pairs. For example, `cpu,host=serverA,region=west usage_system` would be one time 
+series from line 1 in the above example. You can generally think of tags as the dimensions on which you can
+query and slice and dice your time series data. However, you can also do that with fields, provided that you
+pay attention to performance and schema design considerations.
+
+With the data in InfluxDB it's important to keep this context of a time series in mind. There are different types 
+of numbers that can be recorded in time series and many operations only make sense when performed on an individual 
+series and then later combined in some other way. Here are some examples of data that is fed in a time series:
+
+### Metrics
+Metrics are regular time series that are values captured at specific intervals of time (like every 10 seconds or 1 
+minute or 1 hour or 1 day). There are different kinds of metrics that are commonly captured
+
+* counter - A value that counts the occurrence of an event. Counters alway increase unless they have 
+            been wrapped or reset. A reset occurs when what ever is doing the counting is restarted in 
+            which case it starts counting again from 0. A wrapped counter is one that has reached the 
+            largest number allowed by the byte representation of the value, in which case it starts over 
+            again at 0. These wraps and resets need to be accounted for when performing operations on counters. 
+            An example of a counter is the number of bytes a network interface has received. 
+* gauge - A value that measures a specific value, which can increase or decrease. An example of a gauge is 
+          taking a temperature reading.
+
+### Events
+Events are single data points in time. This could be an individual request to an API and how long that took, a user 
+logging into a service, a car parking, a light turning on, a container starting or stopping, or anything you can think 
+of. Events are what are called irregular time series given that they can occur at any time. It is possible to create 
+regular time series from an underlying event stream (like counting how many occurred every five minutes for the last hour).
+
+### Working with data in Flux
+When working with data in Flux, it translates the line protocol/time series structure into to a table. You 
+can think of a measurement name as a table with the tag and field keys representing columns. Flux also 
+includes the reserved column, `time`, in every table pulled from InfluxDB. The `time` column is the time at the end 
+of the line protocol when data is written in.
+
+Here's an example of line protocol and its translation to a Flux table:
+
+```
+h2o_temperature,location=santa_monica,state=CA surface_degrees=65.2,bottom_degrees=50.4 1568756160
```

Comment:
Maybe it would be easier to use a dummy dataset, where you don't have to memorize the tags and fields?  myMeas, myTag_1=A,myTag_2=a myField_1=int,myField_2=float 1